azurerm/resource_arm_eventhub_namespace_disaster_recovery_config.go

```
@@ -0,0 +1,319 @@
+			return fmt.Errorf("Error waiting for replication to compelte for EventHub Namespace Disaster Recovery Configs %q (Namespace %q / Resource Group %q): %s", name, namespaceName, resourceGroup, err)
+		}
+	}
+
+	return resourceArmEventHubNamespaceDisasterRecoveryConfigRead(d, meta)
+}
+
+func resourceArmEventHubNamespaceDisasterRecoveryConfigRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ArmClient).eventhub.DisasterRecoveryConfigsClient
+	ctx := meta.(*ArmClient).StopContext
+
+	id, err := azure.ParseAzureResourceID(d.Id())
+	if err != nil {
+		return err
+	}
+
+	name := id.Path["disasterRecoveryConfigs"]
+	resourceGroup := id.ResourceGroup
+	namespaceName := id.Path["namespaces"]
+
+	resp, err := client.Get(ctx, resourceGroup, namespaceName, name)
+	if err != nil {
+		if utils.ResponseWasNotFound(resp.Response) {
+			d.SetId("")
+			return nil
+		}
+		return fmt.Errorf("Error making Read request on Azure EventHub Namespace Disaster Recovery Configs %q (Namespace %q / Resource Group %q): %s", name, namespaceName, resourceGroup, err)
+	}
+
+	d.Set("name", name)
+	d.Set("namespace_name", namespaceName)
+	d.Set("resource_group_name", resourceGroup)
+
+	if properties := resp.ArmDisasterRecoveryProperties; properties != nil {
+		d.Set("partner_namespace_id", properties.PartnerNamespace)
+		d.Set("alternate_name", properties.AlternateName)
+	}
+
+	return nil
+}
+
+func resourceArmEventHubNamespaceDisasterRecoveryConfigDelete(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ArmClient).eventhub.DisasterRecoveryConfigsClient
+	ctx := meta.(*ArmClient).StopContext
+
+	id, err := azure.ParseAzureResourceID(d.Id())
+	if err != nil {
+		return err
+	}
+
+	name := id.Path["disasterRecoveryConfigs"]
+	resourceGroup := id.ResourceGroup
+	namespaceName := id.Path["namespaces"]
+
+	breakPair, err := client.BreakPairing(ctx, resourceGroup, namespaceName, name)
+	if breakPair.StatusCode != http.StatusOK {
```

(same here) can we raise this error here?